src/strategies/frax-finance/note.txt

```diff
@@ -0,0 +1,7 @@
+TO TEST 
+npm install
+OPTIONAL
+ADD ,
+        "snapshot": '11833575' // change block number
+TO examples.json
+npm run test
```

Comment:
Can you remove this file? You can use "npm run test --strategy=frax-finance" to test your strategy. You don't need to do changes in the files that are not part of the strategy.

Reply:
done